README.md

```diff
@@ -76,15 +75,9 @@ yarn watch
 `yarn watch` will keep monitoring and compiling the babel files, so open
 another terminal to run the tests:
 
-(Note: before you can run the tests, you'll need to configure your PostgreSQL
-server to support logical decoding for our live queries tests. See
-[the @graphile/lds README](packages/lds/README.md#postgresql-configuration) for how
-to enable `wal_level = logical`.)
```

Comment:
Why was this paragraph removed?

Reply:
This seemed like a duplication of information, and not to fit in the light overview ("quick start") of the README. The prerequisites necessary to run the tests are documented extensively in the linked CONTRIBUTING now, which seemed like the better place for this paragraph.

Reply:
I think we should bring it back, otherwise 95% of the people who try the quickstart will fail and not know why. We could simplify it:

```
(Note: your PostgreSQL server must be
[configured for logical decoding](packages/lds/README.md#postgresql-configuration).)
```

Reply:
@ab-pm Just a nudge that this PR can be merged after this small change.